Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -268,6 +282,7 @@
 				FFC8CDA32AA385E800D129A6 /* GoogleService-Info.plist */,
 				FF5D13A62A86C2D600BC9BD6 /* BasicCarMaintenanceApp.swift */,
 				FFBFE08F2A98EFDD000A9BEB /* Models */,
+				8AEE81702ACF383200FC0C2A /* Main */,
```

Comment:
Can you remove the `Main` group, and instead put the `MainTabView` file right under the `BasicCarMaintenanceApp` file?